plutus-core/generators/Language/PlutusCore/Generators/NEAT/Spec.hs

```diff
@@ -377,9 +446,11 @@ packAssertion f t a =
 
 -- | generate examples using `search'` and then generate one big test
 -- that applies the given test to each of them.
+
+-- TODO : printing the number of tests appears to put garbage in the log
 bigTest :: (Check t a, Enumerable a)
         => String -> GenOptions -> t -> (t -> a -> Assertion) -> TestTree
-bigTest s GenOptions{..} t f = testCase s $ do
```

Comment:
I think this attempt to put print the number of terms generated into the log may not be safe -  I think it might be putting lots of blank lines and warnings in the log before finally printing the number.

Reply:
I have changed my mind about this. My log seems to have an arbitrary number of blank lines in it anyway whether I print out the number of generated programs or not - not sure if this is just on my machine.